be/src/vec/exec/scan/new_olap_scanner.cpp

```diff
@@ -412,7 +412,7 @@ Status NewOlapScanner::_init_tablet_reader_params(
         }
     }
 
-    if (!config::disable_storage_page_cache) {
+    if (_state->enable_page_cache()) {
         _tablet_reader_params.use_page_cache = true;
     }
```

Comment:
```cpp
_tablet_reader_params.use_page_cache = _state->enable_page_cache();
```

Reply:
already modifed，Indeed it should, add be to check the configuration passed by fe